includes/Dashboard.php

```diff
@@ -151,7 +151,7 @@ public function load_stories_dashboard() {
 		// TODO Preload templates.
 		$preload_paths = [
 			'/wp/v2/settings',
-			'/web-stories/v1/web-story?context=edit&order=desc&orderby=modified&page=1&per_page=24&status=publish%2Cdraft&_web_stories_envelope=true',
+			'/web-stories/v1/web-story?_embed=author&context=edit&order=desc&orderby=modified&page=1&per_page=24&status=publish%2Cdraft%2Cfuture&_web_stories_envelope=true',
```

Comment:
```suggestion
			'/web-stories/v1/web-story?_embed=1&context=edit&order=desc&orderby=modified&page=1&per_page=24&status=publish%2Cdraft%2Cfuture&_web_stories_envelope=true',
```

Reply:
Author being passed here doesnt do anything in more testing

Reply:
See https://xwp.slack.com/archives/C013J2EGH8R/p1601567869057300?thread_ts=1601567500.056700&cid=C013J2EGH8R to how this should/could be fixed